api/src/main/java/org/openmrs/validator/ConceptSourceValidator.java

```diff
@@ -52,7 +52,8 @@ public void validate(Object obj, Errors errors) throws IllegalArgumentException
 			        + ConceptSource.class);
 		} else {
 			ValidationUtils.rejectIfEmptyOrWhitespace(errors, "name", "error.name");
-			ValidateUtil.validateFieldLengths(errors, obj.getClass(), "name", "hl7Code", "description", "retireReason");
+			ValidateUtil.validateFieldLengths(errors, obj.getClass(), "name", "hl7Code", "uniqueId", "description",
```

Comment:
Nope. Just means if you exceed it max length its going to fail. Like retire  reason. Not needed at creation. But if more than max not allowed